babel.config.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
         ],
     ],
     env: {
-        dev: {
+        development: {
```

Comment:
changed from dev, since its more common to use development, which is also what storybooks defaults use for mode